gnomad/utils/vep.py

```diff
@@ -860,19 +861,21 @@ def filter_vep_transcript_csqs(
 
 
 def filter_vep_transcript_csqs_expr(
-    csq_expr: hl.expr.ArrayExpression,
+    csq_expr: Union[hl.expr.StructExpression, hl.expr.ArrayExpression],
```

Comment:
You add struct, considering if it's exploded?

Reply:
well, it can be used in the way we do for the toolbox because the annotation we are filtering on is top-level, or if it's exploded. Really any situation where the annotation you want to filter on isn't in a list